packages/handlersjs-http/lib/services/sync.service.ts

```diff
@@ -47,7 +45,17 @@ export class SyncService<T, S extends string, P extends string, M extends {
 
     await this.store.set(this.storage, new Set([ ...storage ?? [], ...fetchedValues.flat() ]) as M[S]);
 
-    return this;
+  }
+
+  canHandle(input: void, intermediateOutput?: never): Observable<boolean> {
+
+    return of(true);
+
+  }
+
+  handle(input: void, intermediateOutput?: never): Observable<void> {
+
+    return from(this.sync());
```

Comment:
Since we are working with observables here anyway, I'd make sync return one as well.